config/routes.rb

```diff
@@ -614,6 +614,10 @@
       resources :plans, only: [:show]
       resources :subscriptions, only: [:create]
 
+      match '/pro/sign_up' => '/user#signin',
```

Comment:
This can be dropped as it doesn't _add_ any functionality.

Reply:
vanity route!